models/user.model.js

```diff
@@ -54,6 +54,10 @@
      healthTrackRecords: {
         type : [mongoose.SchemaTypes.ObjectId],
         ref : "HealthTrackRecord"
+     },
+     appointments: {
```

Comment:
Improve this to support the case, where doctor can visit another doctor when he/she is sick